db/database.go

```diff
@@ -81,7 +81,8 @@ type DatabaseContextOptions struct {
 	RevisionCacheCapacity uint32
 	AdminInterface        *string
 	UnsupportedOptions    *UnsupportedOptions
-	TrackDocs             bool // Whether doc tracking channel should be created (used for autoImport, shadowing)
+	AutoImport            bool // Whether autoImport of documents without _sync is enabled
```

Comment:
Is `AutoImport` documented more anywhere?  Is `AutoImport` related to bucket shadowing?